### DISCUSSION TITLE
Is there any plan for Remix to have a traditional forum?

### DISCUSSION THREAD

**Discussion by depsimon** [remix-run/remix#7575] [General]
A good old phpBB for old time's sake!

Nah but seriously though, am I the only one to wish for a web-based forum for Remix?

Next.js has recently mirrored their Discord forum to https://nextjs-forum.com. We can't interact directly with the content so it's nice but still far from ideal.

Is anything similar in the pipelines?

**Comment by sergiodxa** [comment]
I think GitHub Discussions is the most similar thing there will be to an open forum.
